src/darwin/Framework/CHIP/MTRDeviceControllerDelegate.h

```diff
@@ -82,6 +83,20 @@ MTR_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4))
     commissioningComplete:(NSError * _Nullable)error
                    nodeID:(NSNumber * _Nullable)nodeID MTR_AVAILABLE(ios(17.0), macos(14.0), watchos(10.0), tvos(17.0));
 
+/**
+ * Notify the delegate when commissioning is completed.
```

Comment:
This should probably document that if it's implemented it will be called and the other methods will not be?

Those other methods should have been documenting that too.